specifyweb/stored_queries/tests.py

```diff
@@ -476,6 +476,29 @@ def test_relationships_in_switch_fields(self):
             query = query.query.add_columns(expr)
             self.assertCountEqual(list(query), [('text 1 value for this accession',), (' text 2 value for this accession role2',)])
 
+    def test_group_by_with_aggregators(self):
+      with models.session_context() as session:
+        prep_col = func.max(getattr(models.Preparation, models.Preparation._id))
+        
```

Comment:
No. Django creates a `test_DATABASE` during testing. You need to connect to that. What you have is trying to connect to actual database. See the backend tests failing, and read up more on django's test databases.

The test_session_context() instead connects to the test database (created by Django). Use it instead. 

So, use 
```
with FormatterAggregatorTests.test_session_context() as session:
```
instead of 
```
with models.session_context() as session:
```

Reply:
👍 ya, thanks, nice catch, forgot to change that when I moved the function.